docs/sha2-256-chunked.md

```diff
@@ -0,0 +1,43 @@
+# sha2-256-chunked
```

Comment:
will this page end up on the docs website somewhere? or will it be accessible only in the repo?

Reply:
In the short term, just in the repo.  I'm not sure what the right long-term doc strategy is.